recipes/zfel/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "zfel" %}
+{% set version = "0.9.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 53405358ee1e392533ac3f8b0257a5281c888b7bd01db2df5eb1640598480ad1
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - zfel
+  requires:
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pytest tests
+
+about:
+  home: https://github.com/zfel/{{ name }}
```

Don't worry about templating--and it's also not correct
```suggestion
  home: https://github.com/slaclab/zfel
```

---

Fixed!